core/src/main/resources/template/simple/dynamic-attributes.ftl

```diff
@@ -21,12 +21,24 @@
 <#if (parameters.dynamicAttributes?? && parameters.dynamicAttributes?size > 0)><#rt/>
 <#assign aKeys = parameters.dynamicAttributes.keySet()><#rt/>
 <#list aKeys as aKey><#rt/>
-  <#assign keyValue = parameters.dynamicAttributes.get(aKey)/>
```

Comment:
I think it wasn't required to delete these lines and add following duplicate new lines. Adding just this new line before here is more brief and should work: `<#if dynamic_attributes_ignore?? && aKey?starts_with(dynamic_attributes_ignore)><#continue></#if>`

Reply:
As far I see `<#continue>` is [deprecated](https://freemarker.apache.org/docs/ref_directive_list.html#ref_list_continue)

Reply:
I see thanks. But still that link's note proposes other solutions which fits here well, e.g. `?filter(`.

Reply:
Done, I had to use `<#function>`